beluga/test/beluga/include/beluga/test/simple_pointcloud_interface.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BELUGA_TEST_STATIC_SIMPLE_POINTCLOUD_INTERFACE_HPP
+#define BELUGA_TEST_STATIC_SIMPLE_POINTCLOUD_INTERFACE_HPP
+
+#include <range/v3/view/iota.hpp>
+#include <sophus/se3.hpp>
+
+namespace beluga::testing {
+
+template <typename T>
+class SparsePointCloud3 {
```

Comment:
@pvela2017 meta: should this be `SimpleSparsePointCloud3`? Some aliases like `SimpleSparsePointCloud3d` and `SimpleSparsePointCloud3f` would be nice too.

Reply:
Changed in 015f1b79441f103c2e55d662d87fa4c144e0ef2c